resources/faq.md

```diff
@@ -0,0 +1,163 @@
+# FAQ
+
+This FAQ is still a work in progress.<br>
+Do not hesitate to ask us your own questions. It will help us improve this page!
+
+## I have never used a search engine before. Can I use MeiliSearch anyway?
+
+Of course! No need to get used with ElasticSearch or Solr to handle MeiliSearch.
```

Comment:
Of course! No knowledge of ElasticSearch or Solr is required to use MeiliSearch.